shared/app.js

```diff
@@ -73,7 +74,9 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
Yes, issue with passing the the variable instead of string. And my understanding was that reason of https://github.com/rendrjs/rendr/commit/cf3b47bfff8e89c5a73cee37e5af9436a55fb22f was to hide it from browserify (by passing variable instead of string), so default router won't be served to the client when custom router exists.

If latter behavior is still desired we need to come up with something that will suite both parties.